physics/docs/pdftxt/GFS_v16_suite.txt

```diff
@@ -4,10 +4,7 @@
 \section gfsv16_suite_overview Overview
 
 Version 16 of the Global Forecast System (GFS) was implemented operationally by the NOAA
-National Centers for Environmental Prediction (NCEP) in 2021. This suite is available for 
-use with the UFS SRW App and with the CCPP SCM.
-
-The GFS_v16 suite uses the parameterizations in the following order:
+National Centers for Environmental Prediction (NCEP) in 2021. The GFS_v16 suite uses the parameterizations in the following order:
```

Comment:
This will require a longer strategic discussion. I am not sure we should put mention of SCM back into the code right after the release because there could (in theory) be another SRW v2.2 release in another month. 
Since suites are the purview of the host, I think the only way to solve this conundrum is to remove the mention of hosts and suites from ccpp-physics documentation.  So I suggest we discuss this in an upcoming group meeting. Or if you want to continue this discussion via GitHub, open a discussion topic.